AK/Stream.cpp

```diff
@@ -5,6 +5,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "AK/StringBuilder.h"
```

Comment:
```suggestion
#include <AK/StringBuilder.h>
```
And clang format will move it to be in alphabetical order

Reply:
Ah! I was fighting with the clang-format before noticing the quotes. Didn't question why I used quotes though :)